description: enable clang specific compiler flag only if the compiler is clang
author: Michael Gilbert <mgilbert@debian.org>

--- a/third_party/swiftshader/src/Reactor/BUILD.gn
+++ b/third_party/swiftshader/src/Reactor/BUILD.gn
@@ -95,7 +95,7 @@ config("swiftshader_subzero_private_conf
         cflags += [ "-Wno-sign-compare" ]
       }
     }
-  } else if (is_linux || is_mac) {
+  } else if (is_clang) {
     cflags += [ "-Wno-macro-redefined" ]
   }
 
